### ISSUE TITLE
get error when trying to use web socket

### ISSUE THREAD

**Issue by vreesie18** [jaggedsoft/php-binance-api#126]
I tried the first web socket example on your page and only get the following error:

Fatal error: Uncaught Error: Call to undefined function Ratchet\Client\connect() in /var/www/xxxxxx.eu/vendor/jaggedsoft/php-binance-api/php-binance-api.php:2121 Stack trace: #0 /var/www/xxxxxx.eu/binance_test_websockets.php(14): Binance\API->miniTicker(Object(Closure)) #1 {main} thrown in /var/www/xxxxxx.eu/vendor/jaggedsoft/php-binance-api/php-binance-api.php on line 2121

This is my code:

	$Filename = "pathtofile/binance.json";

	require '/var/www/xxxxxx.eu/vendor/jaggedsoft/php-binance-api/php-binance-api.php';
	$api = new Binance\API($Filename);	

	$api->miniTicker(function($api, $ticker) {
		print_r($ticker);
	});

Is there something I do wrong ???

**Comment by jaggedsoft** [comment]
Yeah the problem here is that you are not requiring it via the composer method

For example you can probably get it to work doing it this way:
```php
require 'vendor/autoload.php';
```
Instead of this way `require 'php-binance-api.php';`
The problem is we need to include the RatchetPHP websocket functions

**Comment by MrDevBC** [comment]
1) Download and install composer:
https://getcomposer.org/download/

2) Create a folder in your C:\ drive call Binance

3) Run command prompt and type:
cd C:\Binance
composer require jaggedsoft/php-binance-api

4) Once complete copy the vendor folder into your project.

This what I did and it work...
